src/components/DataView/DataView.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import './DataView.css';
-import Card from '../Card/Card';
+import Card from '../Card/Card.js'
 
-const DataView = () => {
-  return (
-    <div className='card-container'>
+
+
+const DataView = ()=>{
+  return(
+    <div>
```

Comment:
This change breaks tests because the className is required for the tests. Also, the removal of the semi-colons kicks back ESLint errors.